swagger-codegen.log

```diff
@@ -0,0 +1,57 @@
+16:14:35.865 [Thread-0] WARN  i.s.c.v.g.go.AbstractGoCodegen - map (reserved word) cannot be used as model name. Renamed to model_map
```

Comment:
Add it to gitignore please

Reply:
I've removed the log from git all together